packages/logger/tsconfig.json

```diff
@@ -12,21 +13,14 @@
     "declaration": true,
 
     "noImplicitAny": false,
-    "allowSyntheticDefaultImports": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
This used to give a lot of problems combined with jest, as where the project would build but then jest wouldn't run tests because of an import and vice-versa. It's fixed now in `ts-jest` so great you noticed to turn this on 👌